src/components/views/rooms/RoomHeader.tsx

```
@@ -446,11 +546,13 @@ function XrpP2P({ props, onFinished }: any): JSX.Element {
     const [env, setEnv] = useState("");
     const [destinationTag, setDestinationTag] = useState<number | string>("");
     const [sourceTag, setSourceTag] = useState<number | string>("");
-
+    const [qrData, setQrData] = useState(null);
+    const [showQRModal, setShowQRModal] = useState(false);
     const [memoId, setMemoId] = useState(0);
     const [value, setValue] = useState(0);
```

ESLint

[error] 'setValue' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)